data/World/Fire Temple MQ.json

```diff
@@ -11,7 +11,25 @@
                     (can_use(Hookshot) and (can_use(Fire_Arrows) or
                         (can_use(Dins_Fire) and 
                             ((damage_multiplier != 'ohko' and damage_multiplier != 'quadruple') or
-                                can_use(Goron_Tunic) or can_use(Bow) or can_use(Longshot))))))"
+                                can_use(Goron_Tunic) or can_use(Bow) or can_use(Longshot))))))",
+            "Fire Temple MQ First Room Pot 1": "True",
+            "Fire Temple MQ First Room Pot 2": "True",
+            "Fire Temple MQ Near Boss Pot 1": "
+                (logic_fewer_tunic_requirements or can_use(Goron_Tunic)) and has_fire_source and ((can_use(Hookshot)) or can_use(Hover_Boots))",
+            "Fire Temple MQ Near Boss Pot 2": "
+                (logic_fewer_tunic_requirements or can_use(Goron_Tunic)) and has_fire_source and ((can_use(Hookshot)) or can_use(Hover_Boots))",
+            "Fire Temple MQ Before Boss Left Crate 1": "
+                (logic_fewer_tunic_requirements or can_use(Goron_Tunic)) and has_fire_source",
+            "Fire Temple MQ Before Boss Left Crate 2": "
```

Comment:
You have the fire mq near boss left crates in logic as child, however child cannot climb around the flame wall in the dungeon entrance. You can do a flame wall skip, and it's not even that hard, but it would have to be a new trick.